CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Changed
+- Remove `vtex workspace create` && `vtex workspace list`
```

Comment:
Isn't it `vtex workspace create` and `vtex workspace info`? Or is it really `vtex workspace list?`